include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -357,15 +357,25 @@ class format_parse : public format_base
      * \tparam container_option_t Must model the seqan3::sequence_container and
      *                            its value_type must be parseable via parse_option_value
      *
+     * \cond DEV
+     * \tparam format_parse_t is needed to make the function "dependent" (i.e. do instantiation in the second phase of
+     *                        two-phase lookup) as the requires clause needs to be able to access the other
+     *                        parse_option_value overloads.
+     * \endcond
+     *
```

Comment:
```suggestion
     * \tparam container_option_t Must model the seqan3::sequence_container and
     *                            its value_type must be parseable via parse_option_value
     * \tparam format_parse_t Needed to make the function "dependent" (i.e. do instantiation in the second phase of
     *                        two-phase lookup) as the requires clause needs to be able to access the other
     *                        parse_option_value overloads.
     *
```

It's already private, so no `cond DEV` needed.